doc/changes/changes_1.0.1.md

```diff
@@ -2,6 +2,12 @@
 
 Code name: Security fix
 
+## Summary
+
+In this release we fixed a security issue. By an unquoted password it was
+possible to execute code on the host running terraform by using a special
+password.
```

Comment:
I'm not sure it's a good idea to describe how to reproduce the issue. I would just mention that it's related to the unquoted password.

Reply:
Well I don't tell how to reproduce. I just tell what could happen. I think this is important information for users so that they can estimate what could have happened and if they need to take further steps.

Reply:
```suggestion
In this release we fixed a security issue. By an unquoted password it was
possible to execute code on the host running terraform.
```